### ISSUE TITLE
Colab Predictions() and References() instances initialization problem

### ISSUE THREAD

**Issue by asnota** [GEM-benchmark/GEM-metrics#99]
I'm trying to run a lib in a Colab environment, following the instructions from the ReadMe:
```!pip install 'gem-metrics[heavy] @ git+https://github.com/GEM-benchmark/GEM-metrics.git'

list_of_predictions = ["The apple is tasty"]
list_of_references = ["The apple was fresh and tasty"]

import gem_metrics

preds = gem_metrics.texts.Predictions(list_of_predictions)
refs = gem_metrics.texts.References(list_of_references)

result = gem_metrics.compute(preds, refs, metrics_list=['bleu', 'rouge']) 
```

However I'm getting this instead of the metrics output:

`[I 220721 10:04:11 __init__:170] Computing BLEU for None...
[I 220721 10:04:11 __init__:170] Computing ROUGE for None..`

Which is preceeded by "None" messages while initialization of the `preds` and `refs` instances:
`[I 220721 10:04:09 texts:54] Loading predictions for None
[I 220721 10:04:09 texts:54] Loading references for None`

**Comment by tuetschek** [comment]
Hi @asnota , sorry that it's confusing, I guess we need to change the printout... but this is normal behavior when you load the data from memory (otherwise a file name would be shown instead of `None`). So in theory, you should get the metrics scores inside `result` even after these confusing printouts – is `result` empty or do you get any other errors?

**Comment by asnota** [comment]
Hi @tuetschek, you are right, the `result` object does contain the information. Could you explain please what "N" stands for and what is supposed to be printed instean of "None" for ref and pred files (the paths to these files if I were reading jsons?) ?
```{'N': 1,
 'bleu': 21.4441,
 'predictions_file': None,
 'references_file': None,
 'rouge1': {'fmeasure': 0.6, 'precision': 0.75, 'recall': 0.5},
 'rouge2': {'fmeasure': 0.25, 'precision': 0.33333, 'recall': 0.2},
 'rougeL': {'fmeasure': 0.6, 'precision': 0.75, 'recall': 0.5},
 'rougeLsum': {'fmeasure': 0.6, 'precision': 0.75, 'recall': 0.5}}
```

**Comment by tuetschek** [comment]
@asnota I'm not sure I can change `"N"` at this point, but maybe I should add more documentation. Would you be able to share the multi-ref data you used or some case where it fails?

**Comment by asnota** [comment]
@tuetschek the multi-ref data is just a list of 2 sentences:
```
import gem_metrics

list_of_predictions = ["The apple is tasty"]
list_of_references = ["The apple was fresh and tasty", "The fruit is an apple"]

preds = gem_metrics.texts.Predictions(list_of_predictions)
refs = gem_metrics.texts.References(list_of_references)

result = gem_metrics.compute(preds, refs, metrics_list=['ngrams'])  # add list of desired metrics here
```

**Comment by tuetschek** [comment]
@asnota Thanks! And apologies for the delay. There were actually two problems. 

1) If you want to supply multiple references, they need to be passed as lists of lists: 
```
list_of_references = [["The apple was fresh and tasty", "The fruit is an apple"]]
```
Note the double square brackets -- there has to be just 1 data instance that has 2 references, otherwise we wouldn't be able to tell which reference belongs to which data instance.

2) There was a small bug that I just addressed in #100.

Please update & have a look if it works for you now :-) .

**Comment by asnota** [comment]
@tuetschek thanks, it works for me too now.
